/usr/local/lib/libvirt.so.2.4: undefined symbol 'IXDR_PUT_INT32'
/usr/local/lib/libvirt.so.2.4: undefined symbol 'IXDR_GET_INT32'

Index: src/util/virxdrdefs.h
--- src/util/virxdrdefs.h.orig
+++ src/util/virxdrdefs.h
@@ -22,6 +22,12 @@
 
 /* The portablexdr implementation lacks IXDR_PUT_U_INT32 and IXDR_GET_U_INT32
  */
+#ifndef IXDR_PUT_INT32
+# define IXDR_PUT_INT32 IXDR_PUT_LONG
+#endif
+#ifndef IXDR_GET_INT32
+# define IXDR_GET_INT32 IXDR_GET_LONG
+#endif
 #ifndef IXDR_PUT_U_INT32
 # define IXDR_PUT_U_INT32 IXDR_PUT_U_LONG
 #endif
